package.json

```diff
@@ -58,9 +58,9 @@
         "@types/react": "16.8.24",
         "@types/react-redux": "7.1.1",
         "eslint": "5.16.0",
-        "eslint-plugin-eigenspace-react": "latest",
-        "eslint-plugin-eigenspace-redux": "latest",
-        "eslint-plugin-eigenspace-script": "latest",
+        "eslint-plugin-eigenspace-react": "file:src/packages/eslint-plugin-eigenspace-react",
```

Comment:
I've just fetched project, tried invoking yarn install and got:
```
error Package "eslint-plugin-eigenspace-script" refers to a non-existing file '"/Users/dsitdikov/Projects/codestyle/src/packages/eslint-plugin-eigenspace-script/dist"'.
Problem understandable, but not comfortable for developing
```

Reply:
Maybe we could add a requirement for installing inner packages in the Getting started section?

Reply:
Done